utils_linux.go

```diff
@@ -332,7 +340,15 @@ func validateProcessSpec(spec *specs.Process) error {
 	return nil
 }
 
-func startContainer(context *cli.Context, spec *specs.Spec, create bool) (int, error) {
+type CtAct uint8
+
+const (
+	CT_ACT_CREATE CtAct = iota + 1
+	CT_ACT_RUN
+	CT_ACT_RESTORE
+)
+
+func startContainer(context *cli.Context, spec *specs.Spec, action CtAct, criuOpts *libcontainer.CriuOpts) (int, error) {
```

Comment:
I don't really like passing `criuOpts` to `startContainer`. I do like the `action` argument though.